### ISSUE TITLE
paper_config values

### ISSUE THREAD

**Issue by josephgiting** [Ali-Meh619/SAMBA#5]
Thank you very much for sharing this excellent stuff. This is a good paper.

Could you kindly explain the following in relation to the paper_config.py please?
"E = 64, H = 64, R = 3, U = 32, K = 3, and
de = 10 for the SAMBA model. To implement the neural networks,
we use PyTorch library [27] and Adam optimizer [28]. The initial
learning rate is set to 0.001. We consider 1500 epochs for training
the models. The batch size is set to 128. We choose L = 5."

As I tried with the default settings from the git repository and the results were not close. Would appreciate if you can point out what else could make a different.

Many thanks and best wishes.

**Comment by Ali-Meh619** [comment]
Hello,
Thank you for your message. Glad that you are interested in our work.
We ran the experiments for 20 different weight initializations, and picked the best results for our model and all the baselines. This is typically done in all time-series prediction papers.
